specs/vendors_spec.rb

```diff
@@ -0,0 +1,2 @@
+require_relative './spec_helper'
+require_relative '../far_mar'
```

Comment:
You can move this line into `spec_helpers` and not have to do it in every spec file.